ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -322,11 +322,47 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin, EKEventViewDele
                     let endDate = Date (timeIntervalSince1970: endDateDateMillisecondsSinceEpoch!.doubleValue / 1000.0)
                     let ekCalendar = self.eventStore.calendar(withIdentifier: calendarId)
                     if ekCalendar != nil {
-                        let predicate = self.eventStore.predicateForEvents(
-                            withStart: startDate,
-                            end: endDate,
-                            calendars: [ekCalendar!])
-                        let ekEvents = self.eventStore.events(matching: predicate)
+                        var ekEvents = [EKEvent]()
+                        let fourYearsInSeconds = 4 * 365 * 24 * 60 * 60
+                        var currentStartDate = startDate
+                        // Adding 4 years to the start date
+                        var currentEndDate = startDate.addingTimeInterval(TimeInterval(fourYearsInSeconds))
+                        while currentEndDate <= endDate {
+                            let rangeSize = currentEndDate.timeIntervalSince(currentStartDate)
+                            let roundedRangeSize = Int(rangeSize / Double(fourYearsInSeconds)) * fourYearsInSeconds
+                            
+                            // debugPrint("Start date of current range: \(currentStartDate)")
+                            // debugPrint("End date of current range: \(currentEndDate.addingTimeInterval(-1))")
+                            // debugPrint("Range size: \(roundedRangeSize / (365 * 24 * 60 * 60)) years\n")
```

Comment:
please completely remove (not just comment out) debug print statements

Reply:
Done